src/clarity-angular/emphasis/alert/alert.module.ts

```diff
@@ -21,6 +21,9 @@ import { ALERT_DIRECTIVES } from "./index";
     ],
     exports: [
         ALERT_DIRECTIVES
+    ],
+    schemas: [
```

Comment:
not exactly sure why we need this? I don't see this being used anywhere in the project. is it needed because users can add their own icons in alerts?